index.src.html

```diff
@@ -61,8 +61,8 @@
 ## Registration (<a>embedded authenticator</a> mode) ## {#registration-embedded}
 
 - On the phone:
-    * User goes to example.com in the browser, and signs in using whatever method they have been using (possibly a legacy
-        method such as a password).
+    * User goes to example.com in the browser and signs in to an existing account using whatever method they have been using
```

Comment:
Second opinion?

IMO "signs in" is the verb here, "to" is the preposition. That would advocate for leaving this as is, unless you're saying that "signs" is the verb and "into" the direction.